### ISSUE TITLE
Settings: unable to select items in the 'new project/layer' dialog with keyboard

### ISSUE THREAD

**Issue by thomasheartman** [enonic/app-contentstudio#2985]
# Issue

In the 'New...' dialog in the settings part of content studio, I'm unable to use the keyboard to select the options. The keyboard only lets me navigate to the cancel button. 

# Expected behavior

I should be able to select the dialog options using a keyboard. Even better, I'd expect to be able to select items in the same way as in the hierarchy: when the dialog pops up, I can start typing to filter the choices and navigate with the keyboard from then on.

# Video

This video demonstrates that there is no way to select the items in the dialog with the keyboard. The dialog steals focus, but I can only select the 'cancel' button.

https://user-images.githubusercontent.com/17786332/111484017-a79c8000-8735-11eb-9079-d33ab0646cdd.mp4
